toml/src/main/kotlin/org/rust/toml/completion/LocalCargoTomlDependencyCompletionProvider.kt

```diff
@@ -65,6 +68,8 @@ class LocalCargoTomlDependencyCompletionProvider : TomlKeyValueCompletionProvide
                 index.toDouble()
             )
         }
-        result.addAllElements(elements)
+        result.withRelevanceSorter(CompletionSorter.emptySorter().weigh(object : LookupElementWeigher("priorityWeigher", true, false) {
```

Comment:
I'd reformat this code to improve readability, maybe like this:
```Kotlin
val weigher = CompletionSorter.emptySorter()
    .weigh(object : LookupElementWeigher("priority", true, false) {
        override fun weigh(element: LookupElement): Double = (element as PrioritizedLookupElement<*>).priority
    })

result.withRelevanceSorter(weigher).addAllElements(elements)
```

Reply:
And maybe we should add `.weigh(RealPrefixMatchingWeigher())` before our weigher
<img width="452" alt="Screenshot 2021-03-26 at 14 38 05" src="https://user-images.githubusercontent.com/4854600/112626170-e66ebc00-8e40-11eb-962a-633db5f0d5d3.png">